torch_xla/core/xla_env_vars.py

```diff
@@ -26,3 +26,5 @@
 PJRT_GPU_ASYNC_CLIENT = 'PJRT_GPU_ASYNC_CLIENT'
 PJRT_DIST_SERVICE_ADDR = 'PJRT_DIST_SERVICE_ADDR'
 LOCAL_RANK = 'LOCAL_RANK'
+RANK = 'RANK'
```

Comment:
nit: are these constants provided by upstream `torch.distributed` somewhere?

Reply:
I did a search but couldn't find it. A lot of places in upstream uses the string literals.